src/components/ArenaProofModal.tsx

```diff
@@ -56,38 +75,40 @@ export default function ArenaProofModal({
   handleClose,
   activeIndex,
   activeMission,
-  activeSprintId,
-  handleChange: propsHandleChange,
-}) {
+  handleChange,
+}: ArenaProofModalProps) {
+  const { register, handleSubmit, reset } = useForm<{ trashTalk: string }>();
```

Comment:
@AleksandrKurilov let's resolve the issue with the other modal PR first, before we address this one (because, it appears that this code is dependent on the hooks refactoring of the NewProofModal, correct?